actions/universal-description-quicklinks.js

```diff
@@ -1,4 +1,3 @@
-const asyncLib = require('async');
```

Comment:
There shouldn't be any changes in this pull request except for the `remove-banners.js` grandchild. Can you make sure that none of these changes affect anything else?